FlowCal/plot.py

```diff
@@ -1047,6 +1288,1460 @@ def hist1d(data_list,
         plt.savefig(savefig, dpi=savefig_dpi)
         plt.close()
 
+_ViolinRegion = collections.namedtuple('_ViolinRegion',
+                                       field_names=('left_side_x',
+                                                    'right_side_x',
+                                                    'y'))
+
+def _plot_single_violin(violin_position,
```

Comment:
Could you write a short docstring for this function? This is not a user-facing function, but we should have something for people looking to modify this function in the future.

Reply:
Done in 5336718ae4f55574909594ccbea607483e9e7e8f.